app/controllers/spree/admin/payments_controller.rb

```diff
@@ -135,7 +135,9 @@ def authorize_stripe_sca_payment
         return unless @payment.payment_method.class == Spree::Gateway::StripeSCA
 
         @payment.authorize!
-        raise Spree::Core::GatewayError, I18n.t('authorization_failure') unless @payment.pending?
+        return if @payment.pending? && @payment.cvv_response_message.nil?
```

Comment:
cvv_response_message s where the redirect URL sent from stripe is stored. see Stripe::AuthorizeResponsePatcher (I think this is a poor hack, I should have created a specific DB field in the payment to store the payment auth redirect path)